lib/retrospec/plugins/v1/plugin/generators/serializers/rspec_dumper.rb

```diff
@@ -26,6 +26,19 @@ def logger
         @logger
       end
 
+      def dump_transform(d)
+        # This is a bit messy, ideally need to send a patch upstream to puppet
+        # https://github.com/puppetlabs/puppet/blob/master/lib/puppet/pops/visitor.rb#L34
```

Comment:
What is this about exactly?  What would this patch look like?

Reply:
the current do_dump object returns hash's represented in the ruby < 1.9 syntax.  this functions wraps that and munges the hash's to look like >= ruby 1.9 syntax